exporter/otlpexporter/config_test.go

```diff
@@ -82,11 +83,7 @@ func TestLoadConfig(t *testing.T) {
 					Timeout:             30 * time.Second,
 				},
 				WriteBufferSize: 512 * 1024,
-				PerRPCAuth: &configgrpc.PerRPCAuthConfig{
-					AuthType:    "bearer",
-					BearerToken: "some-token",
-				},
-				BalancerName: "round_robin",
```

Comment:
Why this change? It is still using the `configgrpc.PerRPCAuthConfig`

Reply:
Ah my bad